docs/tutorial_carray_memory_profile.ipynb

```diff
@@ -255,10 +255,9 @@
     "mw.stop_watching_memory()\n",
     "\n",
     "d = arange_results\n",
-    "print d\n",
+    "print(d)\n",
     "\n",
-    "arange_results_df = pd.DataFrame({'MiB': arange_results.values()}, index=arange_results.keys())\n",
-    "pd.options.display.mpl_style = 'default'\n",
```

Comment:
removal of this line`"pd.options.display.mpl_style = 'default'\n"` accomplish point 2.